tests/unit/core/TestThemeInstall.js

```diff
@@ -1,35 +1,8 @@
 import { createInternalAddon } from 'core/reducers/addons';
-import themeAction, { getThemeData } from 'core/themePreview';
-import { THEME_PREVIEW } from 'core/constants';
+import { getThemeData } from 'core/themeInstall';
 import { fakeTheme } from 'tests/unit/amo/helpers';
-import { createFakeEvent } from 'tests/unit/helpers';
 
 describe(__filename, () => {
-  it('throws for invalid action', () => {
-    expect(() => {
-      themeAction(null, 'whatever');
-    }).toThrowError('Invalid theme action requested');
-  });
-
-  it('sets-up the event for previews', () => {
-    const fakeNode = {
-      dispatchEvent: sinon.stub(),
-    };
-    const fakeEvent = createFakeEvent({
-      initEvent: sinon.stub(),
-    });
-    const fakeDoc = {
-      createEvent: sinon.stub(),
-    };
-    fakeDoc.createEvent.returns(fakeEvent);
-
-    themeAction(fakeNode, THEME_PREVIEW, fakeDoc);
```

Comment:
You replaced `themeAction` with `themeInstall`, so shouldn't we now have a test similar to this for `themeInstall`?